docs/theory/dft/euler_lagrange_equation.md

```diff
@@ -0,0 +1,98 @@
+## Euler-Lagrange equation
+The fundamental expression in classical density functional theory is the relation between the grand potential $\Omega$ and the intrinsic Helmholtz energy $F$.
+
+$$
+\Omega(T,\mu,[\rho(r)])=F(T,[\rho(r)])-\sum_i\int\rho_i(r)\left(\mu_i-V_i^\mathrm{ext}(r)\right)dr
+$$
+
+What makes this expression so appealing is that the intrinsic Helmholtz energy does only depend on the temperature $T$ and the density profiles $\rho_i(r)$ of the system and not on the external potential $V_i^\mathrm{ext}$.
+
+For a given temperature $T$, chemical potentials $\mu$ and external potentials $V^\mathrm{ext}(r)$ the grand potential reaches a minimum at equilibrium. Mathematically this condition can be written as
+
+$$\left.\frac{\delta\Omega}{\delta\rho_i(r)}\right|_{T,\mu}=F_{\rho_i}(r)-\mu_i+V_i^{\mathrm{ext}}(r)=0\tag{1}$$
+
+where $F_{\rho_i}(r)=\left.\frac{\delta F}{\delta\rho_i(r)}\right|_T$ is short for the functional derivative of the intrinsic Helmholtz energy. In this context, eq. (1) is commonly referred to as the Euler-Lagrange equation, an implicit nonlinear integral equation which needs to be solved for the density profiles of the system.
+
+For a homogeneous (bulk) system, $V^\mathrm{ext}=0$ and we get
+
+$$F_{\rho_i}^\mathrm{b}-\mu_i=0$$
+
+which can be inserted into (1) to give
+
+$$F_{\rho_i}(r)=F_{\rho_i}^\mathrm{b}-V_i^\mathrm{ext}(r)\tag{2}$$
+
+### Spherical molecules
+In the simplest case, the molecules under considerations can be described as spherical. Then the Helmholtz energy can be split in to an ideal and a residual part:
+
+$$\beta F=\sum_i\int\rho_i(r)\left(\ln\left(\rho_i(r)\Lambda_i^3\right)-1\right)dr+\beta F^\mathrm{res}$$
```

Comment:
Maybe use square brackets?

`$$\beta F=\sum_i\int\rho_i(r)\left[\ln\left(\rho_i(r)\Lambda_i^3\right)-1\right]dr+\beta F^\mathrm{res}$$`

Reply:
In this context square brackets are already used to specify a functional dependency.